Description: wmcdplay: FTBFS on hurd-i386
 Hello,
 .
 wmcdplay currently FTBFS on hurd-i386, due to inclusion of a linux-specific
 header <linux/cdrom.h>. The attached patch fixes it by including
 <sys/cdrom.h> instead, could you please apply it?
 .
 Thanks,
 Samuel
Author: Samuel Thibault <sthibault@debian.org>
Bug-Debian: http://bugs.debian.org/647964
Last-Update: 2014-07-29

--- a/cdctl.h
+++ b/cdctl.h
@@ -30,7 +30,12 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <unistd.h>
+#ifdef __linux__
 #include <linux/cdrom.h>
+#endif
+#ifdef __GNU__
+#include <sys/cdrom.h>
+#endif
 
 // CD status values
 #define ssData     0
